LuaUI/Widgets/unit_global_build_command.lua

```diff
@@ -1259,6 +1259,36 @@ function widget:CommandNotify(id, params, options, isZkMex, isAreaMex)
 						myCmd = {id=id, target=target, x=x, y=y, z=z, assignedUnits={}}
 					else -- if the target is a unit
 						myCmd = {id=id, target=target, assignedUnits={}}
+
+						-- Cancel inverse jobs
+						if id == CMD_RECLAIM then
+							-- A reclaim command on a target cancels jobs to repair, resurect, or build that target.
+							local inverseHash
+							inverseHash = BuildHash({id=CMD_REPAIR, target=target})
```

Comment:
Lua VM needlessly assigns `nil` implicitly, even if you initialize it in the next line.
```suggestion
							local inverseHash = BuildHash({id=CMD_REPAIR, target=target})
```